### PR TITLE
W-14120045 open-beta-ga branch (for Oct) compared to latest branch

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/acb-components.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+= Core Processors 
+
+Configure message and event processing in an integration application or API implementation. Core processors (also called Mule components) run on the Mule runtime engine. 
+
```
Comment (dukesphere) [inline modules/ROOT/pages/acb-components.adoc:29]: TODO: We'll need to comment out any that we can't cover in time for ga, TBD.

Comment (dukesphere) [reply modules/ROOT/pages/acb-components.adoc:29]: TODO: also we'll need to add this file to the TOC. @hannanelson Let's discuss plan for this ref in TOC, pls.

Comment (hannanelson) [reply modules/ROOT/pages/acb-components.adoc:29]: Is this reference content? If so, suggest adding to the end with other referencey topics.

**Review by hannanelson — DISMISSED** [review]
LGTM!

**Review by priyaayyar — APPROVED** [review]

**Pull request merged**